random-beans-randomizers/src/main/java/io/github/benas/randombeans/randomizers/FakerBasedRandomizer.java

```diff
@@ -40,11 +40,11 @@
     protected Faker faker;
 
     protected FakerBasedRandomizer() {
-        faker = new Faker();
+        faker = new Faker(Locale.ENGLISH);
```

Comment:
There is not difference between `faker = new Faker();` and `faker = new Faker(Locale.ENGLISH);`, but I think it's better to make it explicit, to show that the used locale is the same as when using a seed.